Tests/Jay/PerformanceTests.swift

```diff
@@ -27,10 +27,14 @@ class PerformanceTests: XCTestCase {
     func urlForFixture(name: String) -> NSURL {
         let url: NSURL
         //if we're running from CLI, use SPM_INSTALL_PATH to find fixtures, otherwise bundle
-        if let path = NSProcessInfo().environment["SPM_INSTALL_PATH"] {
-            url = NSURL(string: "file://\(path)/../Tests/Jay/Fixtures/\(name).json")!
```

Comment:
Can you try to use `#path` as the path to this file to find the fixture path (parent folder...)? I'm not a big fan of using the system call to C to figure it out like this. Sorry I haven't noticed this in my earlier reviews. :)